docker-compose.yml

```diff
@@ -79,12 +79,13 @@ services:
       POSTGRES_DATABASE: search
 
   search-es:
-    image: elasticsearch:2
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.3.1
```

Comment:
I propose to use the same ES image like in the case of snoop-stats-es:
image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.2.2
We can update the snoop-stats-es image or to change search-es image

Reply:
Makes sense. I’m not sure what the stats index is used for. Perhaps it can be upgraded without any trouble.